samcli/commands/pipeline/bootstrap/cli.py

```diff
@@ -113,7 +116,12 @@
     help="Choose a permissions provider to assume the pipeline execution role. Default is to use an IAM User.",
 )
 @click.option(
-    "--oidc-provider-url", help="The URL of the OIDC provider. Example: https://server.example.com", required=False
+    "--oidc-provider-url",
+    help="The URL of the OIDC provider. "
+    "Github Actions: https://token.actions.githubusercontent.com "
+    "GitLab: https//:<GITLAB INSTANCE> "
+    "Bitbucket: https://bitbucket.org/<WORKSPACE>/<REPOSITORY>/admin/addon/admin/pipelines/openid-connect",
```

Comment:
Not sure how useful this is. Terminals can be small, and this would likely add a lot of noise to the help message. Also if these are examples of reasonable defaults, it should just default to them in the prompts.